pkg/pgclient/config.go

```diff
@@ -69,15 +70,16 @@ func ParseFlags(fs *flag.FlagSet, cfg *Config) *Config {
 	fs.DurationVar(&cfg.DbConnectionTimeout, "db-connection-timeout", defaultConnectionTime, "Timeout for establishing the connection between Promscale and TimescaleDB.")
 	fs.BoolVar(&cfg.AsyncAcks, "async-acks", false, "Acknowledge asynchronous inserts. If this is true, the inserter will not wait after insertion of metric data in the database. This increases throughput at the cost of a small chance of data loss.")
 	fs.IntVar(&cfg.ReportInterval, "tput-report", 0, "Interval in seconds at which throughput should be reported.")
-	fs.Uint64Var(&cfg.LabelsCacheSize, "labels-cache-size", 10000, "Maximum number of labels to cache.")
-	fs.Uint64Var(&cfg.MetricsCacheSize, "metrics-cache-size", cache.DefaultMetricCacheSize, "Maximum number of metric names to cache.")
-	fs.Uint64Var(&cfg.SeriesCacheSize, "series-cache-size", cache.DefaultSeriesCacheSize, "Maximum number of series to cache.")
```

Comment:
Since we are refactoring these, we should refactor the `docs/cli.md` to a separate section of **cache**.